### ISSUE TITLE
Monitoring of links and API changes

### ISSUE THREAD

**Issue by caldwellst** [OCHA-DAP/ocha-anticipy#192]
Might be useful to setup CI to (weekly? daily?) make simple calls to some of the APIs we use just to make sure they are still live and working as intended, including ensuring hardcoded links like for CODABs remain active. Raising this now because yesterday [Nepal's CODABs were updated](https://data.humdata.org/dataset/cod-ab-npl), and the individual districts file has been aligned to be adm3, so updating in the CODAB PR, but wondering how long we otherwise would've taken to notice!

**Comment by turnerm** [comment]
Yes exactly, this is what I had in mind with #127. Do you think we should add this to 1.1?
